src/domain/app/footer/Footer.tsx

```diff
@@ -46,17 +48,29 @@ const Footer: React.FC = () => {
       url: ROUTES.ADMIN,
     },
     { labelKey: 'navigation.tabs.help', url: ROUTES.HELP, target: '_self' },
+    {
+      labelKey: 'navigation.tabs.dataProtection',
+      url: DATA_PROTECTION_URL,
+      externalUrl: true,
+    },
   ].filter((i) => i) as NavigationItem[];
```

Comment:
you can change this to `.filter(skipFalsyType)` and after that you can remove NavigationItem interface which is only used to type cast here